src/updater/updater.cpp

```diff
@@ -120,10 +120,16 @@ auto mmapFile(const std::string &filename, boost::interprocess::mode_t mode)
     using boost::interprocess::file_mapping;
     using boost::interprocess::mapped_region;
 
+    {
+        storage::io::FileReader file(filename, storage::io::FileReader::VerifyFingerprint);
```

Comment:
I was wrong anyway, commented out the file reader fingerprint verification and errors still occur. I'll skip fingerprinting these mmap'ed files for now (just one file) to get this to land and open a ticket.